app/src/main/java/com/android/solvit/seeker/ui/provider/ProviderInfo.kt

```diff
@@ -388,50 +384,53 @@ fun ProviderHeader(provider: Provider) {
                       modifier = Modifier.testTag("providerName"),
                       color = colorScheme.onBackground)
                   Text(
-                      text = provider.companyName,
+                      text =
+                          provider.service
+                              .toString()
+                              .lowercase()
+                              .replaceFirstChar { it.titlecaseChar() }
+                              .replace('_', ' '),
```

Comment:
I believe that we have a function `Services.format(service: Services)` that would be helpful for that!